plugins/Jobs.cpp

```diff
@@ -721,19 +721,22 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
     }
 
     // ----------------------------------------------------------------------
-    else if (SIEquals(requestVerb, "RetryJob") || SIEquals(requestVerb, "FinishJob")) {
-        // - RetryJob( jobID, delay, [data] )
+    else if (SIEquals(requestVerb, "RequeueJob") || SIEquals(requestVerb, "RetryJob") || SIEquals(requestVerb, "FinishJob")) {
+        // - RetryJob is deprecated, use RequeueJob
```

Comment:
Beyond semantics, is there a benefit to renaming this command? This will eventually turn into a breaking change and not that we have millions of people using bedrock but something to keep in mind now that this is open source and given that we don't have versioning. This is not a blocker

Reply:
There is no reason beyond semantics

Reply:
There's a lot less overhead to stick with `RetryJob`, I'm going to go that route